src/subcommand/wallet/mint.rs

```diff
@@ -32,6 +37,8 @@ impl Mint {
       bail!("rune {rune} has not been etched");
     };
 
+    let postage = self.postage.unwrap_or(TARGET_POSTAGE);
```

Comment:
If the postage is beneath the dust limit it should return an errors. Something like this:

```
ensure!(
  destination.script_pubkey().dust_limit < postage,
  "postage below dust limit"
);
```